packages/ecr-credential-provider/Cargo.lock

```diff
@@ -0,0 +1,7 @@
+# This file is automatically @generated by Cargo.
```

Comment:
This file isn't necessary.

Reply:
Not sure how it was generated before, but I removed it and ran `cargo update` and it was not recreated. So I've removed it.

Reply:
These `Cargo.lock` files can show up if you run `cargo make` in the package directory rather than in the root.

Reply:
Removed file and cleaned up commit message.